algoliasearch-common/src/main/java/com/algolia/search/Defaults.java

```diff
@@ -19,7 +20,10 @@
 
   ObjectMapper DEFAULT_OBJECT_MAPPER =
       new ObjectMapper()
+          .registerModule(new JavaTimeModule())
           .enable(OBJECT_MAPPER_DEFAULT_FEATURE)
+          .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+          .enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
```

Comment:
Could you add some comments/docs here, why enabling & disabling those?

Reply:
Fixed with eeb6b15! TLDR;

`DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY`: Enabled to allow Jackson to deserialize List/Array of one element.

`SerializationFeature.WRITE_DATES_AS_TIMESTAMPS`: Disabled because most of the dates we **send** to the API are ISO String.

However, those can be overridden with annotations!

Reply:
> `SerializationFeature.WRITE_DATES_AS_TIMESTAMPS`: Disabled because most of the dates we **send** to the API are ISO String.

Since JSON doesn't support date, we should always expect either a string (but we should not do any parsing whatsovever) or an integer I think.

Reply:
So, OK for converting them to timestamp I think